### ISSUE TITLE
Dalton2020

### ISSUE THREAD

**Issue by alserba** [ubccr/software-layer#378]
Hello, I would like to request Dalton2020 build.
Software can be found at:
https://gitlab.com/dalton/dalton/tree/release/2020
Thank you!

**Comment by tonykew** [comment]
Dalton version 2020.1 has been built, and is going through internal testing.
I will update this request once Dalton has been published.

**Comment by tonykew** [comment]
Dalton version 2020.1 has been published to the ccrsoft/2023.01 software release

```
login1$ module spider dalton

----------------------------------------------------------------------------
  dalton: dalton/2020.1
----------------------------------------------------------------------------
    Description:
      The Dalton code is a powerful tool for a wide range of molecular
      properties at different levels of theory. Any published work arising
      from use of one of the Dalton2016 programs must acknowledge that by a
      proper reference, https://www.daltonprogram.org/www/citation.html.

    Properties:
      Chemistry libraries/apps

    You will need to load all module(s) on any one of the lines below before the "dalton/2020.1" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
login1$ module load gcc/11.2.0 openmpi/4.1.1 dalton/2020.1 ucx/1.13.1
-------------------------------------------------------------------------------
The following dependent module(s) are not currently loaded: ucx/1.11.2 (required by: openmpi/4.1.1)
-------------------------------------------------------------------------------
login1$ 
```

Note that I loaded the latest version of UCX to support our newer nodes.
the above warning can be ignored.

This module makes both "dalton" and "dalton.x" available:

```
login1$ ls ${EBROOTDALTON}/bin
dalton  dalton.x
login1$ 
login1$ dalton --help
Unrecognized option: --help

Usage: /cvmfs/soft.ccr.buffalo.edu/versions/2023.01/easybuild/software/avx512/MPI/gcc/11.2.0/openmpi/4.1.1/dalton/2020.1/bin/dalton [options] dalinp{.dal} [molinp{.mol} [potinp{.pot}] [pcmsolver{.pcm}]]

Options:
[...]
login1$ 

```
